onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -552,7 +552,11 @@ def _where_input_wrangler(
     TorchLibOpInfo("exp2", core_ops.aten_exp2),
     TorchLibOpInfo("expand", core_ops.aten_expand),
     TorchLibOpInfo("expand_as", core_ops.aten_expand_as),
-    TorchLibOpInfo("erf", core_ops.aten_erf),
+    TorchLibOpInfo("erf", special_ops.aten_special_erf),
+    TorchLibOpInfo(
+        "erfc", special_ops.aten_special_erfc, tolerance={torch.float16: (1e-2, 2e-4)}
+    ),
+    # TorchLibOpInfo("erfcx", special_ops.aten_special_erfcx),  # not in OPS_DB
```

Comment:
Should we create something in extra_opinfo.py for erfcx?

Reply:
https://github.com/microsoft/onnxscript/issues/825